Error during completion: 'Namespace' object has no attribute 'api_key'
Error during completion: 'Namespace' object has no attribute 'api_key'
Error during completion: 'Namespace' object has no attribute 'api_key'
Error during completion: 'Namespace' object has no attribute 'api_key'
Error during completion: 'Namespace' object has no attribute 'api_key'
Error during completion: 'Namespace' object has no attribute 'api_key'
Error during completion: 'Namespace' object has no attribute 'api_key'
Error during completion: 'Namespace' object has no attribute 'api_key'
Error during completion: 'Namespace' object has no attribute 'api_key'
Error during completion: 'Namespace' object has no attribute 'api_key'
Error during completion: 'Namespace' object has no attribute 'api_key'
Error during completion: 'Namespace' object has no attribute 'api_key'
Error during completion: 'Namespace' object has no attribute 'api_key'
Error during completion: 'Namespace' object has no attribute 'api_key'
Error during completion: 'Namespace' object has no attribute 'api_key'
Error during completion: litellm.BadRequestError: LLM Provider NOT provided. Pass in the LLM provider you are trying to call. You passed model=mistralai/pixtral-large-latest
 Pass model as E.g. For 'Huggingface' inference endpoints pass in `completion(model='huggingface/starcoder',..)` Learn more: https://docs.litellm.ai/docs/providers
Error during completion: litellm.BadRequestError: LLM Provider NOT provided. Pass in the LLM provider you are trying to call. You passed model=mistralai/pixtral-large-latest
 Pass model as E.g. For 'Huggingface' inference endpoints pass in `completion(model='huggingface/starcoder',..)` Learn more: https://docs.litellm.ai/docs/providers
Error during completion: litellm.BadRequestError: LLM Provider NOT provided. Pass in the LLM provider you are trying to call. You passed model=mistralai/pixtral-large-latest
 Pass model as E.g. For 'Huggingface' inference endpoints pass in `completion(model='huggingface/starcoder',..)` Learn more: https://docs.litellm.ai/docs/providers
Error during completion: litellm.BadRequestError: LLM Provider NOT provided. Pass in the LLM provider you are trying to call. You passed model=mistralai/pixtral-large-latest
 Pass model as E.g. For 'Huggingface' inference endpoints pass in `completion(model='huggingface/starcoder',..)` Learn more: https://docs.litellm.ai/docs/providers
Error during completion: litellm.BadRequestError: LLM Provider NOT provided. Pass in the LLM provider you are trying to call. You passed model=mistralai/pixtral-large-latest
 Pass model as E.g. For 'Huggingface' inference endpoints pass in `completion(model='huggingface/starcoder',..)` Learn more: https://docs.litellm.ai/docs/providers
Error during completion: litellm.BadRequestError: LLM Provider NOT provided. Pass in the LLM provider you are trying to call. You passed model=mistralai/pixtral-large-latest
 Pass model as E.g. For 'Huggingface' inference endpoints pass in `completion(model='huggingface/starcoder',..)` Learn more: https://docs.litellm.ai/docs/providers
Error during completion: litellm.BadRequestError: LLM Provider NOT provided. Pass in the LLM provider you are trying to call. You passed model=mistralai/pixtral-large-latest
 Pass model as E.g. For 'Huggingface' inference endpoints pass in `completion(model='huggingface/starcoder',..)` Learn more: https://docs.litellm.ai/docs/providers
Error during completion: litellm.BadRequestError: LLM Provider NOT provided. Pass in the LLM provider you are trying to call. You passed model=mistralai/pixtral-large-latest
 Pass model as E.g. For 'Huggingface' inference endpoints pass in `completion(model='huggingface/starcoder',..)` Learn more: https://docs.litellm.ai/docs/providers
Error during completion: litellm.BadRequestError: LLM Provider NOT provided. Pass in the LLM provider you are trying to call. You passed model=mistralai/pixtral-large-latest
 Pass model as E.g. For 'Huggingface' inference endpoints pass in `completion(model='huggingface/starcoder',..)` Learn more: https://docs.litellm.ai/docs/providers
Error during completion: litellm.BadRequestError: LLM Provider NOT provided. Pass in the LLM provider you are trying to call. You passed model=mistralai/pixtral-large-latest
 Pass model as E.g. For 'Huggingface' inference endpoints pass in `completion(model='huggingface/starcoder',..)` Learn more: https://docs.litellm.ai/docs/providers
Error during completion: litellm.AuthenticationError: AuthenticationError: MistralException - Error code: 401 - {'message': 'Unauthorized', 'request_id': '0bf3f02dd936b641873be77c58ec5167'}
Error during completion: litellm.AuthenticationError: AuthenticationError: MistralException - Error code: 401 - {'message': 'Unauthorized', 'request_id': '489924053a68b9797c63dc25f1d7550e'}
Error during completion: litellm.AuthenticationError: AuthenticationError: MistralException - Error code: 401 - {'message': 'Unauthorized', 'request_id': 'f37960399b22380f7ad94b778cd18215'}
Error during completion: litellm.AuthenticationError: AuthenticationError: MistralException - Error code: 401 - {'message': 'Unauthorized', 'request_id': '729e3f49be03505b4c5b1218f1f6c96e'}
Error during completion: litellm.AuthenticationError: AuthenticationError: MistralException - Error code: 401 - {'message': 'Unauthorized', 'request_id': '313eac3d10974c5bad847f64de253d2a'}
Error during completion: litellm.AuthenticationError: AuthenticationError: MistralException - Error code: 401 - {'message': 'Unauthorized', 'request_id': '15b549300ccbd531449c7b753b307d5c'}
Error during completion: litellm.AuthenticationError: AuthenticationError: MistralException - Error code: 401 - {'message': 'Unauthorized', 'request_id': '2455e21909349b00faaedb01ff5bc516'}
Error during completion: litellm.AuthenticationError: AuthenticationError: MistralException - Error code: 401 - {'message': 'Unauthorized', 'request_id': 'e06c0b291c69ef4263bb6c69b090526b'}
Error during completion: litellm.AuthenticationError: AuthenticationError: MistralException - Error code: 401 - {'message': 'Unauthorized', 'request_id': 'e7dd666d13c5b703d8ff9c756bdcc117'}
Error during completion: litellm.AuthenticationError: AuthenticationError: MistralException - Error code: 401 - {'message': 'Unauthorized', 'request_id': 'c40b1c10822e4b70f9a5965fa2c91338'}
Error during completion: litellm.InternalServerError: The model is repeating the same chunk = ins.
